_data/video.yml

```diff
@@ -3,3 +3,8 @@
   title: About FOSSASIA
   img: FOSSASIA.jpeg
   author_name: Supun Tharinda Edirisuriya
+  
+- url: https://www.youtube.com/watch?v=Noy7OnKIhXM&t=19s
```

Comment:
Correct link - https://www.youtube.com/watch?v=Noy7OnKIhXM
The link you gave transitions to t=19 seconds.